content/en/blog/_posts/2023-08-15-kubernetes-1.28-blog.md

```diff
@@ -0,0 +1,295 @@
+not be able to create replacement Pods. When the kubelet on the shut-down node is not available to delete
+the old pods, an associated StatefulSet cannot create a new pod (which would have the same name).
+
+There's also a problem with storage. If there are volumes used by the pods, existing VolumeAttachments will
+not be disassociated from the original - and now shut down - node so the PersistentVolumes used by these
+pods cannot be attached to a different, healthy node. As a result, an application running on an
+affected StatefulSet may not be able to function properly. If the original, shut down node does come up, then
+their pods will be deleted by its kubelet and new pods can be created on a different running node.
+If the original node does not come up (common with an [immutable infrastructure](https://glossary.cncf.io/immutable-infrastructure/) design),  those pods would be stuck in a `Terminating` status on the shut-down node forever.
+
+For more information on how to trigger cleanup after a non-graceful node shutdown,
+read [non-graceful node shutdown](/docs/concepts/architecture/nodes/#non-graceful-node-shutdown).
+    
+## Improvements to CustomResourceDefinition validation rules 
+
+The [Common Expression Language (CEL)](https://github.com/google/cel-go) can be used to validate
+[custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/). The primary goal is to allow the majority of the validation use cases that might once have needed you, as a CustomResourceDefinition (CRD) author, to design and implement a webhook. Instead, and as a beta feature, you can add _validation expressions_ directly into the schema of a CRD.
+    
+CRDs need direct support for non-trivial validation. While admission webhooks do support CRDs validation, they significantly complicate the development and operability of CRDs.
+
+For more information, read [validation rules](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#validation-rules) in the CRD documentation.
+
+## ValidatingAdmissionPolicies graduate to beta
+    
+Common Expression language for admission control is customizable, in-process validation of requests to the Kubernetes API server as an alternative to validating admission webhooks.
+
+This builds on the capabilities of the CRD Validation Rules feature that graduated to beta in 1.25 but with a focus on the policy enforcement capabilities of validating admission control.
+    
+This will lower the infrastructure barrier to enforcing customizable policies as well as providing primitives that help the community establish and adhere to the best practices of both K8s and its extensions.
+
+To use [ValidatingAdmissionPolicies](/docs/reference/access-authn-authz/validating-admission-policy/), you need to enable the `admissionregistration.k8s.io/v1beta1` API group in your cluster's control plane.
```

Comment:
```
you need to enable both the `admissionregistration.k8s.io/v1beta1` API group and `ValidatingAdmissionPolicy` feature gate in your cluster's control plane.
```
cc @gracenng Since this info is critical, is it possible to edit the blog to address this?